test/__helpers/e2e-system-test.ts

```diff
@@ -101,4 +101,30 @@ export async function e2eTestApp(ctx: ReturnType<typeof setupE2EContext>) {
 
   expect(res.data).toContain('success')
   expect(res.exitCode).toStrictEqual(0)
+
+  // Run built app and query graphql api
+  await ctx.spawn('node', ['node_modules/.build'], async (data, proc) => {
```

Comment:
Could you maybe use the `DEFAULT_BUILD_FOLDER_NAME` constant instead? Or do we want that test to fail if we ever change that constant?

Reply:
If you use that constant, it might we worth to move that constant out of `lib/layout` and put it under `lib/build`